packages/nocms-plugin-directory-resources/src/createScriptManager.js

```diff
@@ -1,15 +1,51 @@
 import fs from 'fs';
+import { rollup  } from 'rollup';
+
+import rollupDefaults from './rollupDefaults';
+
+function trimNewLines(val) {
+	return val
+		.replace(/^\n+/, '')
+		.replace(/\n+$/, '');
+}
+
+let cache;
+async function rollupWithCache(options) {
+	const bundle = await rollup({
+		cache,
+		...options
+	});
+	cache = bundle.cache;
+	return bundle;
+}
 
 export default function createScriptManager() {
-	const scriptsFiles = new Set();
+	const scriptFiles = new Set();
+	const transpileScriptFiles = new Set();
+
+	function registerScript(scriptFile, transpileFlag) {
+		if(transpileFlag === true) {
```

Comment:
@jonatanpedersen using `registerScript`now needs to have a `transpileFlag=true` if rollup should be used like:

```
:register-script(src='./script.js' transpileFlag=true)
```

which brings the performance back to the usual level, since we skip the bundling step